apps/wallet/src/ui/app/pages/home/tokens/TokensDetails.tsx

```diff
@@ -48,9 +48,9 @@ function MyTokens() {
             {balance?.length ? (
                 <div className="flex flex-1 justify-start flex-col w-full mt-6">
                     <Text variant="caption" color="steel" weight="semibold">
-                        MY COINS
+                        My Coins
                     </Text>
-                    <div className="flex flex-col w-full justify-center divide-y divide-solid divide-steel/20 divide-x-0 px-1.5">
+                    <div className="flex flex-col w-full justify-start divide-y divide-solid divide-steel/20 divide-x-0 px-1.5 pb-1">
```

Comment:
Hmm, I don't see how this fixes the issue - changing the justification doesn't do anything (I don't think the `div` container needs to be flexed in the first place) and 4px of bottom padding doesn't seem to do much either. The wallet doesn't seem to be scrollable if I have three coins in my wallet (pic 1), and if I have five coins then the fifth coin is hidden behind the tab container (pic 2). LMK if I'm misunderstanding something 😄 

<img width="356" alt="image" src="https://user-images.githubusercontent.com/7453188/234650392-79e3b453-9b6d-445a-94b1-63ca2ddc0766.png">

<img width="1245" alt="image" src="https://user-images.githubusercontent.com/7453188/234650927-892318a3-a05e-4be1-8f28-3a52b20a3481.png">

Reply:
my bad, should not have change it